### PR TITLE
[serializer][ObjectNormalizer] fixed Undefined attributeName.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Branch | master |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | ~ |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

File: src/Symfony/Component/Serializer/Normalizer/ObjectNormalizer.php [hunk 1]
```diff
@@ -55,9 +55,8 @@ protected function extractAttributes($object, $format = null, array $context = a
             ) {
                 continue;
             }
-
             $name = $reflMethod->name;
-
+            $attributeName = false;
```
Comment (xabbuh) [inline src/Symfony/Component/Serializer/Normalizer/ObjectNormalizer.php]: I think `null` as the default value would be better.

File: src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php [hunk 2]
```diff
@@ -508,6 +508,11 @@ class ObjectDummy
     protected $camelCase;
     protected $object;
 
+    public function otherMethod()
+    {
+        throw new \RuntimeException('Dummy::otherMethod() should not be called');
+    }
+
```
Comment (xabbuh) [inline src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php]: Is this change necessary? Otherwise we should avoid it.

File: src/Symfony/Component/Serializer/Normalizer/ObjectNormalizer.php (hunk 1 above)
Comment (aitboudad) [inline src/Symfony/Component/Serializer/Normalizer/ObjectNormalizer.php]: done

**Comment by xabbuh** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @aitboudad.
